docs/draft/notifications.md

```diff
@@ -0,0 +1,49 @@
+---
+layout: default
+title: Notifications
+nav_order: 2
+has_children: true
+permalink: /specs/draft
+---
+
+# Notifications in Status
+
+## Why is there no Push Notifications?
+
+Push Notifications, as offered by Apple and Google are a privacy concern, they require a centralized service that is aware of who the notification needs to be delivered to.
+
+An early implementation of push notifications was done using the Firebase library which required users to exchange notifications ids and send notifications through Firebase. This was fairly limited by the fact that one could only send/receive a notification after exchanging these ids with the contact. We were also not satisfied with the fact that it was a huge dependency, which required the installation of `Play Services` that some users weren't comfortable with. 
+
+## Local Notifications
```

Comment:
This is a better start. I'd start by describing requirements and design goals. Then lead to how we solve it with local notifications for Android, and write about limitation for iOS.